server/src/schema.js

```diff
@@ -45,6 +45,7 @@ export const Schema = [
     status: String!
     startTime: Int!
     endTime: Int!
+    userId: Int
```

Comment:
I was wondering how long before we needed to do something like this. We are almost at the point of needing some level of user permission or system to allow one user to act on behalf of another.